plugin.program.remote.control.browser/addon.xml

```diff
@@ -0,0 +1,32 @@
+        <description lang="pt_BR">A forma mais poderosa de acessar o conteúdo no Netflix e YouTube e Amazon Instant Video seria um navegador web, se navegadores web fornecessem um bom suporte nativo para a interface de usuário de 10-foot. Este add-on inicia um navegador e conecta os botões de navigação do controle remoto para o ponteiro do mouse. Esta é a interface mais amigável para consumir conteúdo online sem a necessidade de um teclado sem fio.</description>
+        <disclaimer lang="en">The experience will be degraded unless these external dependencies are installed: “psutil,” “pyalsaaudio,” “pylirc2,” and “Pillow.” Another helpful utility is “unclutter,” which automatically hides the mouse pointer. (On a Debian-based system, run “sudo apt-get install python-psutil python-alsaaudio python-pylirc python-pil unclutter”). Finally, a theme with a large mouse pointer will improve pointer visibility, (e.g., https://www.gnome-look.org/p/999574/).</disclaimer>
+        <disclaimer lang="en_US">The experience will be degraded unless these external dependencies are installed: “psutil,” “pyalsaaudio,” “pylirc2,” and “Pillow.” Another helpful utility is “unclutter,” which automatically hides the mouse pointer. (On a Debian-based system, run “sudo apt-get install python-psutil python-alsaaudio python-pylirc python-pil unclutter”). Finally, a theme with a large mouse pointer will improve pointer visibility, (e.g., https://www.gnome-look.org/p/999574/).</disclaimer>
+        <disclaimer lang="de_DE">Die Erfahrung wird vermindert werden, wenn nicht diese externen Abhängigkeiten installiert sind: „psutil“, „pyalsaaudio“, „pylirc2“ und „Pillow“. Ein weiteres hilfreiches Werkzeug ist „unclutter“, das automatisch den Mauszeiger versteckt. (Auf einem Debian-basierten System führen Sie „sudo apt-get install python-psutil python-alsaaudio python-pylirc python-pil unclutter“ aus). Schließlich wird ein Theme mit einem großen Mauszeiger die Sichtbarkeit verbessern, (zum Beispiel https://www.gnome-look.org/p/999574/).</disclaimer>
+        <disclaimer lang="pt_BR">A experiência será degradada, a menos que essas dependências externas estejam instaladas: “psutil”, “pyalsaaudio”, “pylirc2” e “Pillow”. Outra ferramenta útil é “unclutter”, que oculta automaticamente o ponteiro do mouse. (Em um sistema baseado em Debian, execute “sudo apt-get install python-psutil python-alsaaudio python-pylirc python-pil unclutter”). Finalmente, um tema com um ponteiro do mouse grande irá melhorar a visibilidade do ponteiro, (por exemplo, https://www.gnome-look.org/p/999574/).</disclaimer>
+        <platform>linux osx</platform>
+        <language>en en_US de_DE pt_BR</language>
```

Comment:
I think that's correct. At least when I first submitted the RTP Play addon I also made the same mistake (english and portuguese as language options due to the addon available translations). I was advised by Martijn (IIRC) to change it to Portuguese only since it was only about the video content.